tests/notebook/test_demo_notebook.ipynb

```diff
@@ -1792,7 +1792,7 @@
     }
    ],
    "source": [
-    "ed_flights.query('Carrier == \"Kibana Airlines\" & AvgTicketPrice > 900.0 & Cancelled == True')"
+    "# NBVAL_IGNORE_OUTPUT\n"
```

Comment:
Let's update this cell with the Pandas 2 output? (And keep the `IGNORE_OUTPUT` with maybe a comment about it if possible.)

Reply:
Done!